helm/crds/gateway.networking.k8s.io_tlsroutes.yaml

```diff
@@ -0,0 +1,894 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
I found we need to add that CRD under the folder `helm/crds/`  ,otherwise the built helm chart don't include that (that's the reason why e2e-test github action failed at the begining)